packages/build-utils/test/fixtures/22-pnpm/vercel.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file can be deleted, right?

Reply:
Unfortunately no, I was getting an undefined error here: https://github.com/vercel/vercel/blob/2a24210b7d3e1412bb6877358bb483c892ad3f72/test/lib/deployment/test-deployment.js#L57-L61 and the test was failing.

Would you prefer a change in that file?